src/main/features/directions-questionnaire/views/hearing-location.njk

```diff
@@ -0,0 +1,77 @@
+            {% elseif party === MadeBy.DEFENDANT %}
+              <p>{{ t('Tell us your preferred location for a hearing, in case the claim goes to one.') }}</p>
+              <p>{{ t('We’ll use the location you choose unless the claimant has an exceptional reason to object, for example accessibility needs.') }}</p>
+              <p>{{ t('You can choose any location - {{ courtName }} is the nearest to the home address you gave us.', { courtName: courtName }) }}</p>
+            {% elseif party === MadeBy.CLAIMANT %}
+              <p>{{ t('You can request any location - {{ courtName }} is the nearest to the home address you gave us.', { courtName: courtName }) }}
+            {% endif %}
+            <h1
+              class="heading-medium">{{ t('Do you want to choose {{ courtName }}', { courtName: form.valueFor('courtName') }) }}</h1>
+            {{ radioGroup(
+              form = form,
+              label = heading,
+              labelForScreenReadersOnly = true,
+              name = 'courtAccepted',
+              options = [
+                {
+                  label: 'Yes',
+                  value: YesNoOption.YES.option,
+                  checked: form.valueFor('courtAccepted') === YesNoOption.YES
+                },
+                {
+                  label: 'No',
+                  value: YesNoOption.NO.option,
+                  checked: form.valueFor('courtAccepted') === YesNoOption.NO
+                }
+              ]
+            ) }}
+            <div class="js-hidden" id="courtAccepted-no">
+              <h1 class="heading-medium">Choose another court</h1>
+              {{ radioOption(form = form, label = 'Enter court name', name = 'alternativeOption', value = 'name', checked = form.valueFor('alternativeOption') === 'name', bold = true) }}
+              <div class="form-group panel panel-border-narrow js-hidden " id="alternativeOption-name" aria-hidden="false">
+                {{ textInput(label = t('Court name'), form = form, name = 'alternativeCourtName', inputClass = 'form-control-1-30') }}
+              </div>
+              {{ radioOption(form = form, label = 'Search by postcode', name = 'alternativeOption', value = 'postcode', checked = form.valueFor('alternativeOption') === 'postcode', bold = true) }}
+              <div class="form-group panel panel-border-narrow js-hidden " id="alternativeOption-postcode" aria-hidden="false">
+                {{ textInput(label = t('Postcode'), form = form, name = 'alternativePostcode', inputClass = 'form-control-1-10') }}
+              </div>
+            </div>
+          {% else %}
+            <h1 class="heading-medium">{{ t('Enter a court name') }}</h1>
+            {{ textInput(form = form, name = 'alternativeCourtName', inputClass = 'form-control-1-100') }}
+          {% endif %}
+          {% if courtName %}
+            <input type="hidden" name="courtName" value="{{ courtName }}">
```

Comment:
Do we have any standard about if these HTML5 optionally-closed tags should be closed?
`<input ... />`

Reply:
The macro 'input' in form.njk of cmc-common-frontend, where the majority of our inputs end up being creating from doesn't close the input tag there. That's probably the closest I can find to us having a standard.